test/Microsoft.AspNetCore.Routing.Tests/Template/TemplateParserTests.cs

```diff
@@ -528,8 +525,8 @@ public void Parse_RegularExpressions_Unescaped(string template)
 
         [Theory]
         [InlineData("{p1}.{p2?}.{p3}", "p2", ".")]
-        [InlineData("{p1?}{p2}", "p1", "p2")]
-        [InlineData("{p1?}{p2?}", "p1", "p2")]
+        [InlineData("{p1?}{p2}", "p1", "{p2}")]
+        [InlineData("{p1?}{p2?}", "p1", "{p2?}")]
```

Comment:
How come you changed these?

Reply:
I changed what gets passed to the error messages. now instead of just including the parameter name or an unrelated thing, it includes the text of the next part..